spec/features/editing_tags/edit_tags_spec.rb

```diff
@@ -17,8 +19,8 @@
   end
 
   def given_there_is_an_edition
-    multi_tag_field = build(:tag_field, type: "multi_tag", id: "multi_tag_id")
-    single_tag_field = build(:tag_field, type: "single_tag", id: "single_tag_id")
+    multi_tag_field = build(:tag_field, type: "multi_tag", id: multi_tag_id)
```

Comment:
I don't think traits would actually help make edit_tags_spec.rb any more readable, as this test suite looks at the HTML of the page and asserts values/IDs. If the ID we're asserting against is actually hidden inside the factory, we'd have two places to update the ID if it ever changes.

However, adding the traits does allow me to simplify some of the other tag tests. I've encapsulated it as a separate refactor commit.

Reply:
Yes, simplifying the tests was what I was thinking of here. I don't think it will make it any more or less readable. While the tests still need to use the IDs, the benefit of this change should be that they don't need to know precisely what they are, since we can just ask the field each time.